config/forestry/backpacks.cfg

```diff
@@ -13391,6 +13391,116 @@ backpacks {
+            dreamcraft:item.CoinChemistIII
+            dreamcraft:item.CoinChemistIV
+            dreamcraft:item.CoinWitch
+            dreamcraft:item.CoinWitchI
+            dreamcraft:item.CoinWitchII
+            dreamcraft:item.CoinWitchIII
+            dreamcraft:item.CoinWitchIV
+            dreamcraft:item.CoinDarkWizard
+            dreamcraft:item.CoinDarkWizardI
+            dreamcraft:item.CoinDarkWizardII
+            dreamcraft:item.CoinDarkWizardIII
+            dreamcraft:item.CoinDarkWizardIV
+            dreamcraft:item.CoinAdventure
+            dreamcraft:item.CoinAdventureI
+            dreamcraft:item.CoinAdventureII
+            dreamcraft:item.CoinAdventureIII
+            dreamcraft:item.CoinAdventureIV
+            dreamcraft:item.CoinSurvivor
+            dreamcraft:item.CoinSurvivorI
+            dreamcraft:item.CoinSurvivorII
+            dreamcraft:item.CoinSurvivorIII
+            dreamcraft:item.CoinSurvivorIV
+            dreamcraft:item.CoinCook
+            dreamcraft:item.CoinCookI
+            dreamcraft:item.CoinCookII
+            dreamcraft:item.CoinCookIII
+            dreamcraft:item.CoinCookIV
+            dreamcraft:item.CoinBees
+            dreamcraft:item.CoinBeesI
+            dreamcraft:item.CoinBeesII
+            dreamcraft:item.CoinBeesIII
+            dreamcraft:item.CoinBeesIV
+            dreamcraft:item.CoinChunkloaderTier
+            dreamcraft:item.CoinChunkloaderTierI
+            dreamcraft:item.CoinChunkloaderTierII
+            dreamcraft:item.CoinChunkloaderTierIII
+            dreamcraft:item.CoinChunkloaderTierIV
+            dreamcraft:item.CoinSmith
+            dreamcraft:item.CoinSmithI
+            dreamcraft:item.CoinSmithII
+            dreamcraft:item.CoinSmithIII
+            dreamcraft:item.CoinSmithIV
+            dreamcraft:item.CoinTechnician
+            dreamcraft:item.CoinTechnicianI
+            dreamcraft:item.CoinTechnicianII
+            dreamcraft:item.CoinTechnicianIII
+            dreamcraft:item.CoinTechnicianIV
+            dreamcraft:item.CoinFlower
+            dreamcraft:item.CoinFlowerI
+            dreamcraft:item.CoinFlowerII
+            dreamcraft:item.CoinFlowerIII
+            dreamcraft:item.CoinFlowerIV
+            dreamcraft:item.CoinBlood
+            dreamcraft:item.CoinBloodI
+            dreamcraft:item.CoinBloodII
+            dreamcraft:item.CoinBloodIII
+            dreamcraft:item.CoinBloodIV
+            dreamcraft:item.CoinSpace
+            dreamcraft:item.CoinSpaceI
+            dreamcraft:item.CoinSpaceII
+            dreamcraft:item.CoinSpaceIII
+            dreamcraft:item.CoinSpaceIV
+            dreamcraft:item.CoinDonation
+            miscutils:itemGenericToken
+            Thaumcraft:ItemResource:18
+            IC2:itemCoin
+            gregtech:gt.metaitem.01:32000
+            gregtech:gt.metaitem.01:32001
+            gregtech:gt.metaitem.01:32002
+            gregtech:gt.metaitem.01:32013
```

Comment:
Item with meta 32013 listed twice. There should be meta 32003, shouldn't it?

Reply:
yea make sense

Reply:
fixed